cmds/ps/ps_linux.go

```diff
@@ -215,34 +202,33 @@ func (p process) getTime() string {
 	return fmt.Sprintf("%02d:%02d:%02d", hrs, mins, secs)
 }
 
-// Walk from the proc files
-// and parsing them
+// Create a ProcessTable containing stats on all processes.
 func (pT *ProcessTable) LoadTable() error {
-	pf := regexp.MustCompile(allProc)
-	err := filepath.Walk(proc, func(name string, fi os.FileInfo, err error) error {
+	// Match all files and directories directly inside of /proc.
+	matches, err := filepath.Glob(filepath.Join(proc, "*"))
+	if err != nil {
+		return err
+	}
+
+	for _, m := range matches {
+		// Filter out files and directories which are not numbers.
+		pid, err := strconv.Atoi(filepath.Base(m))
```

Comment:
here you turn it to an int and above you turn it to a string. Might be simpler to have a function or use regexp to just match numeric strings.